docs/en/modules/net_info.md

```diff
@@ -0,0 +1,79 @@
+# net_info Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-01-21 | [smcl](http://blog.mclemon.io/esp8266-contributing-to-the-nodemcu-ecosystem) | [wolfgangr](https://github.com/wolfgangr/nodemcu-firmware) | [net_info.c](../../../app/modules/net_info.c)
```

Comment:
Whether you want to put GitHub username "smcl" there or "Sean McLemon" is up to you. I personally see no reason to stick to the username - unless you want to link his GitHub account rather than his blog. Same applies to yourself of course.
We never formalized it anywhere but in this scenario Sean is the _origin_ and you are _contributor_ i.e. the one who eventually contributed code to NodeMCU.